utils/src/ooinstall/openshift_ansible.py

```diff
@@ -11,30 +11,46 @@
 
 CFG = None
 
+ROLES_TO_GROUPS_MAP = {
+    'master': 'masters',
+    'node': 'nodes',
+    'storage': 'nfs'
```

Comment:
As long as the implementation handles roles/groups that are not defined in the installer, I think we'll be good.

For example, I think it would be perfectly valid to break out etcd hosts separately using the arbitrary config, where we would not support that through the guided installer.